src/main/java/Point.java

```diff
@@ -0,0 +1,17 @@
+public class Point {
+    public double x;
+    public double y;
+
+    public Point(double x, double y){
+        this.x = x;
+        this.y = y;
+    }
+
+    /** 
+     *  @Param a point 
```

Comment:
Could note that it is the euclidan meteric that is used.

Reply:
Fixed